Tests/IntegrationTests/Roslyn/RoslynGenTests.cs

```diff
@@ -102,23 +188,42 @@ private IPackageManager CreatePackageManager(string rootDir)
             return mgr;
         }
 
-        private IPackage AddPackage(IPackageManager manager, string id, string version, string payloadAssemblyFilePath)
+        private IPackage AddPackage(IPackageManager manager, string id, string version, string payloadAssemblyFilePath, params IPackage[] dependencies)
```

Comment:
Both _RoslynGenTests_ and _AnalyzerPluginGeneratorTests_ seem to test the same class. There was previously only one test in _RoslynGenTests_, so it would make sense to consolidate it with the other test class.